onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -171,6 +142,10 @@ def create_backend_test(testname=None):
                                       'test_unsqueeze_negative_axes_cpu',
                                       'test_constant_pad_cpu']
 
+        if c2.supports_device('MKLDNN'):
+            current_failing_tests += ['^test_range_float_type_positive_delta_expanded_cpu',
+                                      '^test_range_int32_type_negative_delta_expanded_cpu']		
+ 									  
         if c2.supports_device('OPENVINO_GPU_FP32') or c2.supports_device('OPENVINO_GPU_FP16'):
```

Comment:
nit: tabs here and at end of previous line

Reply:
Fixed.